README.md

```diff
@@ -33,7 +33,7 @@ Key features and advantages:
 - Query and model in the same language; everything is reusable and extensible.
 - Malloy reads the schema so you don’t need to model everything: Malloy allows creation of re-usable metrics and logic, but there’s no need for boilerplate code that doesn’t add anything new.
 - Pipelining: output one query into the next easily for powerful advanced analysis.
-- Aggregating Subqueries (AKA Turtles) let you delve deeper into data quickly, and nest data infinitely.
+- Aggregating Subqueries let you delve deeper into data quickly, and nest data infinitely.
```

Comment:
Aggregating Subqueries (AKA Turtles) let you build nested data sets  to delve deeper into data quickly, and return complicated networks of data from single queries (like GraphQL).